torch/_dynamo/optimizations/distributed.py

```diff
@@ -41,13 +42,20 @@ def pretty_print_buckets(buckets: List[Bucket]):
         print("Please `pip install tabulate` in order to pretty-print ddp bucket sizes")
 
 
+def fx_name(fqn):
```

Comment:
This looks a bit fragile. Is this a standard thing in pytorch? Could you add a comment pointtin to the code that defines this format?

Reply:
I guess let's make this explicitly private? _fx_name?